packages/playground/src/components/pull-request/pull-request-compare-button.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuGroup,
+  DropdownMenuItem,
+  DropdownMenuTrigger
+} from '@harnessio/canary'
+import { Icon, Text, Button, ButtonGroup } from '@harnessio/canary'
+import { FormFields } from '../../layouts/SandboxPullRequestCompareLayout'
+
+interface PullRequestCompareButtonProps {
+  isSubmitted: boolean
+  isValid: boolean
+  isLoading: boolean
+  formRef: React.RefObject<HTMLFormElement>
```

Comment:
Do we need to use formRef in each form we had in gitness currently? I'm developing a form for it so curious about it

Reply:
not necessarily, i just have a formRef because i need something in my child component that is used in the parent @jessieyu777